x-team/auto-ui
# Creating and editing note experience should be the same

**Release Type:** *Non-Breaking Feature* 


Fixes https://x-team-internal.atlassian.net/browse/XP-2518

## Description

We decided to follow recently with the path where we will align Notes experience which will only show us the plain textarea editor. 

## Checklist

**Before submitting a pull request,** please make sure the following is done:



## Steps to Test or Reproduce

1. Go into applicant
2. Add a new note (you can use markdown syntax)
3. Edit recent note, you should see only the plain textarea, when you select a word you should not see any context menu from Medium
4. Save 


## Impacted Areas in Application

Notes Inline editing, NotesFeed

## Screenshots

![zrzut ekranu z 2018-12-18 12-23-30](https://user-images.githubusercontent.com/39776/50150927-c5ad7880-02bf-11e9-8045-480eb1058801.png)
